fails:Hash#initialize is private
fails:Hash#initialize returns self
fails:Hash#initialize raises a RuntimeError if called on a frozen instance
